whisper-AI-exec-script/action-transcribe-event.xml

```diff
@@ -0,0 +1,240 @@
+            <input id="mLarge" name="transcriptionModel" type="radio" class="configField" value="LARGE">
+	          <label for="mLarge">Large (Speed ~1x) [Disabled]</label>
+          </li>
+        <ul>
+        <legend style="color:seagreen">Automatic translation (Do not add for events in english) </legend>
+        <br>
+          <li>
+            <input id="transEn" name="translEn" type="checkbox" class="configField" value="false" />
+	          <label for="transEn">Add automatic transcription to english</label>
+          </li>
+        <!-- Place to keep the layoutType state --> 
+          <!-- Place to keep the layoutType state --> 
+          <input type="hidden" id="layoutType" class="configField" value="NONE" />
+      </fieldset>
+
+      <script>
+          document.getElementById("mTiny").disabled = true;
+          document.getElementById("mSmall").disabled = true;
+          document.getElementById("mLarge").disabled = true;
+
+
+      </script>
+      
+     </div>
+
+  ]]>
+
+  </configuration_panel>
+  
+  <operations>
+  <state-mappings>
+    <state-mapping state="running">EVENTS.EVENTS.STATE_MAPPING.PUBLISHING</state-mapping>
+    <state-mapping state="failing">EVENTS.EVENTS.STATE_MAPPING.PUBLISHING</state-mapping>
+  </state-mappings>
+
+
+    <!-- Set translation parameters-->
+
+    <operation
+      id="conditional-config"
+      exception-handler-workflow="partial-error"
+      description="Setting transcription variables">
+      <configurations>
+        <configuration key="configuration-name">transcription-model</configuration>
+        <configuration key="condition-1">${mTiny}</configuration>
+        <configuration key="value-1">{{ whisper_server }}:9000</configuration>
+        <configuration key="condition-2">${modelBase}</configuration>
+        <configuration key="value-2">{{ whisper_server }}:9001</configuration>
+        <configuration key="condition-3">${mSmall}</configuration>
+        <configuration key="value-3">{{ whisper_server }}:9002</configuration>
+        <configuration key="condition-4">${mMedium}</configuration>
+        <configuration key="value-4">{{ whisper_server }}:9003</configuration>
+        <configuration key="condition-5">${mLarge}</configuration>
+        <configuration key="value-5">{{ whisper_server }}:9004</configuration>
+        <configuration key="no-match">no-model</configuration>
+ 
+      </configurations>
+    </operation>
+
+    <!-- Analyze media assets -->
+
+    <operation
+      id="analyze-tracks"
+      exception-handler-workflow="partial-error"
+      description="|__   Analyze tracks in media package and set control variables">
+      <configurations>
+        <configuration key="source-flavor">*/source</configuration>
+      </configurations>
+    </operation>
+
+    <!-- Delete old subtitles --> 
+
+     <operation
+      id="tag"
+      exception-handler-workflow="partial-error"
+      fail-on-error="false"
+      description="Remove old archived subtitles ">
```

Comment:
Warning about that in the READM would be great.

Reply:
Added from your another suggestion.